firestore.rules

```diff
@@ -36,6 +36,9 @@ service cloud.firestore {
       allow update: if false;
       allow delete: if false;
     }
+    match /whitelists/{whitelistId} {
+        allow read:   if isSignedIn() && (didSign() || isAdmin()); //TODO (see below #2)
```

Comment:
What are the default permissions if not specified? Do we need to explicitelly set false for write, update, and delete?

Reply:
the default permission is `false` (meaning no access)
couldn't find it in the docs but that's how it behaves in my test environment